net/netxng/src/TNetXNGFile.cxx

```diff
@@ -132,6 +132,11 @@ TNetXNGFile::TNetXNGFile(const char *url,
 {
    using namespace XrdCl;
 
+   // Set the log level
+   TString val = gEnv->GetValue("NetXNG.Debug",     "");
+   if (val.IsNull()) val = gSystem->Getenv("XRD_LOGLEVEL");
```

Comment:
I'd assume that an env var is overriding any setting in rootrc, not the inverse?

Reply:
Right. I have uploaded the change. Thanks, G